src/app/find-help/transport-search/transport-search.component.html

```diff
@@ -0,0 +1,23 @@
+<mat-card>
+  <app-transport-search-form (search)="search($event)"></app-transport-search-form>
+</mat-card>
+<div class="results" *ngIf="this.total && this.total >= 0" [class.loading]="loading">
+  <div class="total">Lista ogłoszeń ({{ total }}):</div>
+  <div>
+    <app-search-result
+      *ngFor="let result of results"
+      [location]="result.origin"
+      [destination]="result.destination"
+      [title]="result.title"
+      [description]="result.description"
+      [attributes]="[
+        { test: !!result.transportDate, text: result.transportDate + '', icon: 'date_range' },
+        {
+          test: !!result.capacity,
+          text: result.capacity + ' ' + this.pluralizePolish('osoba', 'osoby', 'osób', result.capacity),
+          icon: 'person'
+        }
+      ]"
```

Comment:
I'm starting to think, my idea of using attributes parameters wasn't a good one. 
We should refactor it to:
```
<app-search-result ....>
 <app-search-attribute icon='date_range' *ngIf="result.transportDate">{{result.transportDate}}</app-search-attribute>
...
</...>```
What do you think?

Reply:
@sfc-gh-krzymkowski makes sense to me, that's much clearer and more idiomatic.

Reply:
Done.